go.mod

```diff
@@ -98,7 +98,7 @@ require (
 	github.com/vito/houdini v1.1.1
 	github.com/vito/twentythousandtonnesofcrudeoil v0.0.0-20180305154709-3b21ad808fcb
 	go.etcd.io/bbolt v1.3.3 // indirect
-	go.opencensus.io v0.22.2
+	go.opencensus.io v0.22.2 // indirect
```

Comment:
@aoldershaw noticed this change 🤔  don't know why it'd become `indirect` now 🤷‍♂  

```console
$ go mod why go.opencensus.io
github.com/concourse/concourse/tracing
go.opentelemetry.io/otel/exporter/trace/stackdriver
cloud.google.com/go/trace/apiv2
cloud.google.com/go/trace/apiv2.test
cloud.google.com/go/internal/testutil
go.opencensus.io/trace
go.opencensus.io/internal
go.opencensus.io
```